docs/getting-started/first-app.md

```diff
@@ -5,70 +5,82 @@ sidebar_label: Your first application
 slug: /first-app
 ---
 
-# Your first application
+# Your First Application - Tutorial 
```

Comment:
@MichaelSolati is working on a demo app with a tutorial that will eventually go to the docs site. 
I think we should create a folder for tutorials and demos. 
so, just to make things more structured let's remove the word "Tutorial' from this doc.